webcompat/static/config_data/labels.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "url": "https://api.github.com/repos/webcompat/web-bugs/labels/status-duplicate",
+    "name": "status-duplicate",
+    "color": "cccccc"
```

Comment:
> Well, it's not exactly hand-written - just downloaded from GitHub :-p

Heh, yeah. But in the future we'll edit it and I'll leave out a quote mark and the app will explode. :bomb: 

Another option is instead of building the file, we *just* download it from GitHub from time to time? Maybe every 24 hours? (labels don't get added that frequently...)

> To build this file on startup, I should add code to webcompat/init.py

Yeah. We've also got some code for doing checksums @ https://github.com/webcompat/webcompat.com/blob/master/webcompat/helpers.py#L58-L68 if you want to go down the conditional builds route.